stixcore/version_conf.py

```diff
@@ -6,7 +6,7 @@ def get_conf_version():
         with open(Path(__file__).parent / "config" / "data" / "common" / "VERSION.TXT") as f:
             return f.readline().strip()
     except Exception:
-        return 'v0.1.3'
+        return 'v0.1.5'
```

Comment:
Would a place holder be better here `0.0.0` as it likely we won't remember to update this?

Reply:
what do you think?

Reply:
integrated